tests/functional/spec/list_collector_section_summary_items.spec.js

```diff
@@ -0,0 +1,139 @@
+import AnyCompaniesOrBranchesDrivingQuestionPage from "../generated_pages/list_collector_section_summary_items/any-companies-or-branches.page.js";
+import AnyCompaniesOrBranchesPage from "../generated_pages/list_collector_section_summary_items/any-other-companies-or-branches.page.js";
+import AnyCompaniesOrBranchesAddPage from "../generated_pages/list_collector_section_summary_items/any-other-companies-or-branches-add.page.js";
+import AnyCompaniesOrBranchesRemovePage from "../generated_pages/list_collector_section_summary_items/any-other-companies-or-branches-remove.page.js";
+import SectionSummaryPage from "../generated_pages/list_collector_section_summary_items/section-companies-summary.page";
+import EstimatePage from "../generated_pages/list_collector_section_summary_items/estimate-checkbox.page";
+
+describe("List Collector Section Summary Items", () => {
+  describe("Given I launch the test list collector section summary items survey", () => {
+    beforeEach(() => {
+      browser.openQuestionnaire("test_list_collector_section_summary_items.json");
+    });
+    it("When I answer yes to the driving question, Then I should be able to add my own item and the relevant data about it and get to the list collector question page.", () => {
+      $(AnyCompaniesOrBranchesDrivingQuestionPage.yes()).click();
+      $(AnyCompaniesOrBranchesDrivingQuestionPage.submit()).click();
+      $(AnyCompaniesOrBranchesAddPage.companyOrBranchName()).setValue("Company A");
+      $(AnyCompaniesOrBranchesAddPage.registrationNumber()).setValue("0123456789");
+      $(AnyCompaniesOrBranchesAddPage.authorisedInsurerRadioYes()).click();
+      $(AnyCompaniesOrBranchesAddPage.submit()).click();
+      expect(browser.getUrl()).to.contain(AnyCompaniesOrBranchesPage.url());
+    });
+    it("When I add my own item and relevant data, Then after submitting I should be asked if I have more items to add.", () => {
+      $(AnyCompaniesOrBranchesDrivingQuestionPage.yes()).click();
+      $(AnyCompaniesOrBranchesDrivingQuestionPage.submit()).click();
+      $(AnyCompaniesOrBranchesAddPage.companyOrBranchName()).setValue("Company A");
+      $(AnyCompaniesOrBranchesAddPage.registrationNumber()).setValue("0123456789");
+      $(AnyCompaniesOrBranchesAddPage.authorisedInsurerRadioYes()).click();
+      $(AnyCompaniesOrBranchesAddPage.submit()).click();
+      expect($(AnyCompaniesOrBranchesPage.yes()).isExisting()).to.be.true;
+      expect($(AnyCompaniesOrBranchesPage.no()).isExisting()).to.be.true;
+      expect($(AnyCompaniesOrBranchesPage.heading()).getText()).to.contain(
+        "Do you need to add any other UK companies or branches that undertake general insurance business?"
+      );
+    });
+    it("When I add my own item and relevant data, Then after I answer no on additional items page I should get to the section summary page.", () => {
```

Comment:
These aren't new. All these should be tested already. You should test that the summary displays a list collector and its related answers as expected.

Reply:
So I modified them so they assert conditions requested in the comment above.